cmd/extensions/main.go

```diff
@@ -138,19 +120,11 @@ func main() {
 		logger.WithError(err).Fatal("Could not create in cluster config")
 	}
 
-	clientConf.QPS = float32(ctlConf.APIServerSustainedQPS)
```

Comment:
QPS/Burst seem relevant for anything that carries a kubeClient .. but I'm wondering if `cloudproduct.Initialize` is the only thing using the kubeClient? If so, you could make it more obvious by separating it into its own block or function, i.e.:

{
  kubeClient, err = ...
  cloudProduct.Initialize()
}

then, especially if we don't need the informers, it makes it clear to the reader that we're not really using the client at all.

Reply:
I thought I fixed this comment after I left the comment about the allocation service, but ignore me about `kubeClient` - I'm fairly sure you need it for that. (Which means definitely keep QPS/Burst.)

Reply:
Added the QPSs back.